repos/spack_repo/builtin/packages/py_torchgeo/package.py

```diff
@@ -18,6 +18,7 @@ class PyTorchgeo(PythonPackage):
     maintainers("adamjstewart", "calebrob6")
```

Comment:
You can also add yourself to the list of maintainers if you want. These people get notified to review PRs when the package recipe is updated.